packages/frontend/app/routes/wallet-addresses.$walletAddressId.tsx

```diff
@@ -148,6 +148,43 @@ export default function ViewAssetPage() {
             </div>
           </div>
         </div>
+        <div className='grid grid-cols-1 py-3 gap-6 md:grid-cols-3 border-b border-pearl'>
+          <div className='col-span-1 pt-3'>
+            <h3 className='text-lg font-medium'>Liquidity Information</h3>
+          </div>
+          <div className='md:col-span-2 bg-white rounded-md shadow-md'>
+            <div className='w-full p-4 flex justify-between items-center'>
+              <div>
+                <p className='font-medium'>Amount</p>
+                <p className='mt-1'>
+                  {formatAmount(
+                    walletAddress.liquidity ?? '0',
+                    walletAddress.asset.scale
+                  )}{' '}
+                  {walletAddress.asset.code}
+                </p>
+              </div>
+              <div className='flex space-x-4'>
+                {BigInt(walletAddress.liquidity ?? '0') ? (
+                  <Button
+                    aria-label='withdraw wallet address liquidity page'
+                    preventScrollReset
+                    to={`/wallet-addresses/${walletAddress.id}/withdraw-liquidity`}
+                  >
+                    Withdraw
+                  </Button>
+                ) : (
```

Comment:
See PR body for more details, but I was unable to test this button which calls the `createWalletAddressWithdrawal` endpoint.

Thoughts on how to do that are welcome. The button does follow the same pattern as incoming/outgoing withdrawals so I think the frontend is wired up correctly at least.

Reply:
Yeah, you would need to send an ILP packet directly to the wallet address... 
You could probably hack this by adding liquidity manually during app startup into the wallet address via `createDeposit` maybe?